### PR TITLE
[onert/test] Revive operator kernel test script

### PR DESCRIPTION
**Pull request by hseok-oh**
This commit updates runtime backend operator kernel test script.

ONE-DCO-1.0-Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

---

Draft: https://github.com/Samsung/ONE/pull/10496
Related issue: https://github.com/Samsung/ONE/issues/9877

### CODE REVIEW

File: tests/scripts/common.sh [hunk 1]
```diff
@@ -68,23 +68,3 @@ function print_with_dots()
     printf '%s' "$MSG"
     printf '%*.*s ' 0 $padlength "$pad"
 }
-
-
-function run_benchmark_and_print()
```
Comment (hseok-oh) [inline tests/scripts/common.sh:73]: Moved into benchmark script: not commonly used

**Comment by glistening** [comment]
From code changes,

- `profile_for_he_shed` seems removed, and only Linear executor will be tested after this PR.

I guess there are more changes. But it will take me more time to figure out. : )

`Revive` seems a typo. Perhaps `Revise`.

**Comment by hseok-oh** [comment]
>`profile_for_he_shed` seems removed, and only Linear executor will be tested after this PR.

This script was designed to be used for general runtime test through NNAPI, but it is used for operator kernel performance test.  

>`Revive` seems a typo

I used `revive` because this script cannot test runtime kernel without this PR (test only tensorflow lite kernel)

**Review by glistening — APPROVED** [review]
Thank you for explanation. This script was going to be removed since nnapi is not our interests any longer. But you revive this script by changing the content to run benchmark. Frankly, I don't fully understand the details. What is the input. Usage says it gets file, but I don't know what is the file. (tflite?) Anyway, LGTM : )

**Pull request merged**